### ISSUE TITLE
 TypeError: Unsplash is not a constructor, when using node and ES6 Modules

### ISSUE THREAD

**Issue by jonasfrey** [unsplash/unsplash-js#126]
**Solution**
```javascript
import Unsplash from "unsplash-js";

const unsplash = new Unsplash.default({ accessKey: "{access_key}" });
const toJson = Unsplash.toJson;
```

**Comment by OliverJAsh** [comment]
Can you try this?

```js
import Unsplash, { toJson } from "unsplash-js";
const unsplash = new Unsplash({ accessKey: "{access_key}" });
```

**Comment by samijaber** [comment]
closed in #143
